Algorithm.CSharp/AddFutureContractWithContinuousRegressionAlgorithm.cs

```diff
@@ -116,7 +116,7 @@ public override void OnSecuritiesChanged(SecurityChanges changes)
         /// <summary>
         /// Data Points count of all timeslices of algorithm
         /// </summary>
-        public long DataPoints => 59;
+        public long DataPoints => 63;
```

Comment:
Not sure it would be recuded in this case, since extended market hours was not enabled for this algorithms. It is slightly increasing because we added a few more data points to ES minute datasets

Reply:
Added a comment in the PR description:

> We see a slight increase in the data points count in regression algorithms like AddFutureContractWithContinuousRegressionAlgorithm. In that case, the algorithm's start and end dates are 2013/10/6 (sunday) and 10, but it is set to run OnData only once, because the securities are removed after the orders are placed. Before, the algorithm ran only in the 6th, but since that day is now extended market, the algorithm runs now until the 7th.